src/persistence/paths.cpp

```diff
@@ -365,11 +365,25 @@ QString Paths::getAppCacheDirPath() const
 #endif
 }
 
+QString Paths::getExampleNodesFilePath() const
+{
+    QDir dir(getSettingsDirPath());
+    constexpr static char nodesFileName[] = "bootstrapNodes.example.json";
```

Comment:
This is a nice solution. Gives someone a reference without preventing us from updating it later.